.github/workflows/docs.yml

```diff
@@ -4,11 +4,11 @@ on:
   push:
     branches: ["main"]
     paths:
-      - "**/*.md"
+      - '**/*.md'
```

Comment:
Double quotes are used in build.yml and that seems to work

Reply:
yeah, that's what i was thinking, but I am not sure why it's not triggering the markdown-lint job for https://github.com/redhat-et/apex/pull/416